pipermerriam/flex
# Feature/api request and response validation

### What is the problem / feature ?

It'd be really cool if you could take a generic response object from an api and the swagger schema, validate the response against the schema.
### How did it get fixed / implemented ?

LOTS of code and Lots of tests
### How can someone test / see it ?

Check the docs for info on how to use this.

_Here is a cute animal picture for your troubles..._

![img_1338](https://cloud.githubusercontent.com/assets/824194/4894494/32438460-63d4-11e4-9b08-4a373ad2ecde.jpg)
